docs/tutorials/4_data_pipeline.md

```diff
@@ -87,6 +87,39 @@ test_pipeline = [
 ]
 ```
 
+We have supported some lazy operators and encourage users to apply them.
+Lazy ops record how the data should be processed, but it will postpone the processing on the raw data until the raw data forward `Fuse` stage.
+Specifically, lazy ops avoid frequent reading and modification operation on the raw data, but process the raw data once in the final Fuse stage, thus accelerating data preprocessing greatly.
```

Comment:
Any numbers for `greatly`?

Reply:
The effect on 320p K400 is obvious (but it could also be the memcache's efffect). I'll remove `greatly` frist. A comparison result may be posted in the future.

Reply:
@dreamerlin Do you remember the experiment？